vnext/Shared/WinRTWebSocketResource.cpp

```diff
@@ -208,20 +205,11 @@ fire_and_forget WinRTWebSocketResource::PerformWrite() noexcept
     }
 
     size_t length;
-    std::pair<string, bool> front;
-    bool popped = self->m_writeQueue.try_pop(front);
-    if (!popped)
-    {
-      throw hresult_error(E_FAIL, L"Could not retrieve outgoing message.");
-    }
-
-    auto [message, isBinary] = std::move(front);
-
     if (isBinary)
     {
       self->m_socket.Control().MessageType(SocketMessageType::Binary);
 
-      auto buffer = CryptographicBuffer::DecodeFromBase64String(Utf8ToUtf16(std::move(message)));
+      auto buffer = CryptographicBuffer::DecodeFromBase64String(Utf8ToUtf16(messageLocal));
```

Comment:
>messageLocal [](start = 76, length = 12)

What does Utf8toUtf16 take? Is it string&?

Reply:
It takes `const std::string &utf8`.